packages/near-fast-auth-signer-e2e-tests/scripts/faucet.js

```diff
@@ -0,0 +1,87 @@
+const { CovalentClient } = require('@covalenthq/client-sdk');
+
+// Same addresses as the ones in constants.ts, they can't be imported here because this script is run by node.js
+const ETH_PERSONAL_KEY_ADDRESS = '0xf64750f13f75fb9e2f4d9fd98ab72d742d1e33eb';
+const BNB_DOMAIN_KEY_ADDRESS =  '0x81d205120a9f04d3f1ce733c5ed0a0bc66714c71';
```

Comment:
wonder if we should move this to secret for github action and for local dev, we should define them inside `.env`?

Reply:
I'm indifferent about this.
But I think we should leave it here,  we can quickly see the use of the addresses from the comment on top of it instead of figuring out what it does an an `.env ` variable.